container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_aggregator_functions/package_private_helpers.go

```diff
@@ -0,0 +1,105 @@
+package logs_aggregator_functions
```

Comment:
nit: I find `package_private_helpers` to be a very non-descriptive name for a file. Can we find something more representative of what it contains?